docs/user-interface/layouts/horizontalstacklayout.md

```diff
@@ -0,0 +1,166 @@
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="StackLayoutDemos.Views.HorizontalStackLayoutPage">
+    <HorizontalStackLayout Margin="20">
+       <Rectangle Fill="Red"
+                  HeightRequest="30"
+                  WidthRequest="30" />
+       <Label Text="Red"
+              FontSize="Large" />
+    </HorizontalStackLayout>
+</ContentPage>
+```
+
+This example creates a `HorizontalStackLayout` containing a `Rectangle` and a `Label` object. By default, there is no space between the child views:
+
+:::image type="content" source="media/horizontalstacklayout/basic.png" alt-text="HorizontalStackLayout displaying two views screenshot.":::
+
+> [!NOTE]
+> The value of the `Margin` property represents the distance between an element and its adjacent elements. <!--For more information, see [Margin and Padding](margin-and-padding.md).-->
+
+## Space between child views
+
+The spacing between child views in a `HorizontalStackLayout` can be changed by setting the `Spacing` property to a `double` value:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="StackLayoutDemos.Views.HorizontalStackLayoutPage">
+    <HorizontalStackLayout Margin="20"
+                           Spacing="10">
+       <Rectangle Fill="Red"
+                  HeightRequest="30"
+                  WidthRequest="30" />
+       <Label Text="Red"
+              FontSize="Large" />
+    </HorizontalStackLayout>
+</ContentPage>
+```
+
+This example creates a `HorizontalStackLayout` containing a `Rectangle` and a `Label` object, that have ten device-independent units of space between them:
+
+:::image type="content" source="media/horizontalstacklayout/spacing.png" alt-text="HorizontalStackLayout displaying two spaced views screenshot.":::
+
+> [!TIP]
+> The `Spacing` property can be set to negative values to make child views overlap.
+
+## Position and size child views
+
+The size and position of child views within a `HorizontalStackLayout` depends upon the values of the child views' `HeightRequest` and `WidthRequest` properties, and the values of their `VerticalOptions` properties. In a `HorizontalStackLayout`, child views expand to fill the available height when their size isn't explicitly set.
+
+The `VerticalOptions` properties of a `HorizontalStackLayout`, and its child views, can be set to fields from the `LayoutOptions` struct, which encapsulates an *alignment* layout preference. This layout preference determines the position and size of a child view within its parent layout.
+
+> [!TIP]
+> Don't set the `VerticalOptions` property of a `HorizontalStackLayout` unless you need to. The default value of `LayoutOptions.Fill` allows for the best layout optimization. Changing this property has a cost and consumes memory, even when setting it back to its default value.
```

Comment:
This is not the case.

Reply:
Is it also not the case for the new StackLayout?